sources/install/package_mobile.sh

```diff
@@ -103,18 +95,20 @@ function install_androguard() {
 function install_mobsf() {
     # CODE-CHECK-WHITELIST=add-aliases
     colorecho "Installing Mobile Security Framework"
-    fapt wkhtmltopdf
+    fapt wkhtmltopdf libxmlsec1 libxmlsec1-dev
     git -C /opt/tools clone --depth 1 https://github.com/MobSF/Mobile-Security-Framework-MobSF MobSF
     cd /opt/tools/MobSF || exit
     # pipx --preinstall git+https://github.com/MobSF/yara-python-dex.git /opt/tools/MobSF would be needed for ARM64
     #  in the mean time, switching to manual venv and an alias for mobsf
-    local temp_fix_limit="2024-11-01"
+    local temp_fix_limit="2025-04-01"
     if [[ "$(date +%Y%m%d)" -gt "$(date -d $temp_fix_limit +%Y%m%d)" ]]; then
       criticalecho "Temp fix expired. Exiting." # check if pipx supports preinstall now
     else
       python3 -m venv --system-site-packages ./venv
       ./venv/bin/python3 -m pip install git+https://github.com/MobSF/yara-python-dex.git
       ./venv/bin/python3 -m pip install .
+      # https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/2503
+      # pip install xmlsec==1.3.14
```

Comment:
`xmlsec` should be downgraded to make mobsf works
See <https://github.com/MobSF/Mobile-Security-Framework-MobSF/issues/2503>

Reply:
done, thank you 👍